README.md

```diff
@@ -1 +1,47 @@
-# tft-match-history
+# TFT-Match-History
+
+A backend service that allows you to get various match and summoner statistics for TeamFightTactics.
```

Comment:
Consider putting a link to the Team Fight Tactics website in case the reader knows nothing about it.